Oleksii_Makarov/HW8/Lesson8_2.py

```diff
@@ -0,0 +1,35 @@
+from typing import Callable
+
+
+def filter_func(callback: Callable, sequence: iter) -> iter:
+    """
+    Returns items from sequence based on conditional callback
+    """
+    return [item for item in sequence if callback(item)]
+
+
+def condition_numbers(number: int|iter) -> bool:
+    """
+    Conditional function
+    """
+    if number < 4.5:
+        return False
+    else:
+        return True
+
+
+def condition_m(word: list) -> bool:
+    """
+    Conditional function
+    """
+    if word.startswith('M'):
+        return True
+    else:
+        return False
```

Comment:
you can just use lambda functions except defining these ones. It is a suggestion for the future, do not remove them right now.